src/storage/btree_load.c

```diff
@@ -4757,13 +4790,18 @@ online_index_builder (THREAD_ENTRY * thread_p, BTID_INT * btid_int, HFID * hfids
   PR_EVAL_FNC filter_eval_fnc;
   DB_TYPE single_node_type = DB_TYPE_NULL;
   int attr_offset;
-  DB_VALUE dbvalue;
   DB_VALUE *p_dbvalue;
   int *p_prefix_length;
+  uint64_t tasks_started = 0;
   char midxkey_buf[DBVAL_BUFSIZE + MAX_ALIGNMENT], *aligned_midxkey_buf;
+  index_builder_loader_context load_context;
+  cubthread::entry_workpool * ib_workpool =
+    cubthread::get_manager ()->create_worker_pool (prm_get_integer_value (PRM_ID_INDEX_BUILDER_THREAD_COUNT), 32,
```

Comment:
Disable indent for statement using namespace prefix. The syntax using `::` confuses it.